flaml/automl/automl.py

```diff
@@ -2816,10 +2816,10 @@ def is_to_reverse_metric(metric, task):
                 starting_points = "data"
             logger.warning(
                 "No search budget is provided via time_budget or max_iter."
-                " Training only one model per estimator."
-                " Zero-shot AutoML is used for certain tasks and estimators."                
-                " To tune hyperparameters for each estimator,"
-                " please provide budget either via time_budget or max_iter."
+                "Training only one model per estimator."
+                "Zero-shot AutoML is used for certain tasks and estimators."
+                "To tune hyperparameters for each estimator,"
+                "please provide budget either via time_budget or max_iter."
```

Comment:
The space in the beginning is needed. Otherwise there won't be spaces between the sentences.

Reply:
I understand. Fixed. Since I can not automatically lint it using pre-commit, I suspect need to remove it.